website/static/css/custom.css

```diff
@@ -195,6 +181,10 @@ html[data-theme='dark'] .args-list > dt {
   content: ': ';
 }
 
+.dl-inline dd {
+  margin: 0;
+}
```

Comment:
Forgot to note this in the summary. This style is to fix this 👇🏼 
![Screen Shot 2021-01-14 at 9 54 04 AM](https://user-images.githubusercontent.com/4202993/104622432-7b3d8800-564e-11eb-8356-ff4f97736478.png)